test/user-profile/helpers.unit.spec.js

```diff
@@ -0,0 +1,90 @@
+import { expect } from 'chai';
+import _ from 'lodash';
+
+import {
+  formTitles,
+  formLinks,
+  handleIncompleteInformation,
+  handleNonSIPEnabledForm,
+  sipEnabledForms,
+} from '../../src/js/user-profile/helpers';
+
+import fullSchema1010ez from '../../src/js/hca/config/form';
+import fullSchema1990 from '../../src/js/edu-benefits/1990-rjsf/config/form';
+import fullSchema1990e from '../../src/js/edu-benefits/1990e/config/form';
+import fullSchema1990n from '../../src/js/edu-benefits/1990n/config/form';
+import fullSchema1995 from '../../src/js/edu-benefits/1995/config/form';
+import fullSchema5490 from '../../src/js/edu-benefits/5490/config/form';
+import fullSchema5495 from '../../src/js/edu-benefits/5495/config/form';
+import fullSchema527EZ from '../../src/js/pensions/config/form';
+import fullSchema530 from '../../src/js/burials/config/form';
+import fullSchema10007 from '../../src/js/pre-need/config/form';
+
+import schemas from 'vets-json-schema/dist/schemas';
+
+// Maps schema id to config id
+const schemaToConfigIds = {
```

Comment:
While the keys on this object are used, the properties are essentially for visibility, as formIds are the basis of the removal logic, yet:
- schema ids and config ids do not align
- not all configs have a formId (e.g. pre-need)